ml-proto/test/stdio_write.wast

```diff
@@ -0,0 +1,14 @@
+(module 
+  (import $write "stdio" "write" (param i32 i32))
+
+  (memory 4096 4096 (segment 0 "hello, world!\0D\0A\00"))
```

Comment:
Which stdlib is going to do `\n` -> `\r\n` translation, the stdlib linked into the wasm app, or the stdlib linked into ocaml? If the former, it doesn't know that it's on Windows, because wasm presents an OS-independent abstraction, so it doesn't know whether it needs to do that or not. If the latter, then that's what I'm asking for, and the testcase here doesn't need its `\r`.